.github/workflows/.deploy.yaml

```diff
@@ -12,12 +12,14 @@ jobs:
   deploy-to-dev-gcp:
     name: dev-gcp
     runs-on: ubuntu-latest
+    permissions:
```

Comment:
Trenger vi å sette permissions her og i hver enkelt app?

Reply:
Ja, her seier vi kva denne "del-jobben" treng, og i kvar app gir vi denne deljobben lov til dette.

Trur kanskje ikkje vi _må_ definere det her, men jobben treng det, så eg liker å ha det eksplisitt med.

Er tilsvarande som for docker-build-push i .build.yaml.

(og ja, eg irriterer meg litt over at vi må definere det dobbelt, men no can do)